tests/testthat/test-generate_eval_data.R

```diff
@@ -17,14 +17,18 @@ test_that(
       oracle_output = oracle_output
     )
 
-    check_exp_scores_for_window(out_path,
-                                "Full season",
-                                model_out_tbl,
-                                oracle_output)
-    check_exp_scores_for_window(out_path,
-                                "Last 5 weeks",
-                                model_out_tbl |> dplyr::filter(reference_date >= "2022-12-17"),
-                                oracle_output)
+    check_exp_scores_for_set(out_path,
+                             "Full season",
+                             model_out_tbl,
+                             oracle_output)
+    check_exp_scores_for_set(out_path,
+                             "Last 5 weeks",
+                             model_out_tbl |> dplyr::filter(reference_date >= "2022-12-17"),
+                             oracle_output)
+    check_exp_scores_for_set(out_path,
```

Comment:
new test check case based on modifying the existing config file at `test_path("testdata", "test_configs", "config_valid_mean_median_quantile.yaml")` to add an evaluation set with filters on horizon and location.